### DISCUSSION TITLE
Could not find template

### DISCUSSION THREAD

**Discussion by meetgyn** [projectdiscovery/nuclei#1030] [Q&A] [answered]
**Describe the bug**
I installed the new nuclei 2.5.1, but when I try to update the templates I get an error message that I can not heal.
Nuclei Update-Templates 
I'm using Kali 2021 

I try to make the update, then I try to run the nuclei -u http://teste.com.br he finds nothing, zero. This happens to any target that I try to use 

**Nuclei version**
version 2.5.1

**Screenshot of the error or bug**
Imagen - error
![image](https://user-images.githubusercontent.com/3695608/133361360-b874e421-a41f-4a30-ba17-a640c6e96c05.png)

Image - config.yaml
![image](https://user-images.githubusercontent.com/3695608/133361419-818633f1-f2e0-4d2a-a468-a3f12b58df0a.png)

**Comment by geeknik** [comment, marked as answer]
`vulnerabilities` is misspelled.

**Comment by meetgyn** [comment]
> 
> 
> `vulnerabilities` is misspelled.

Now that you said I could identify the error. I was very nervous yesterday and it hurt my reasoning. Thanks for your help.

**Comment by geeknik** [comment]
You’re welcome! Good luck out there! \m/
